Modules/System Test Libraries/GeoLocation/src/GeoLocationTestLibrary.Codeunit.al

```diff
@@ -0,0 +1,46 @@
+// ------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+// ------------------------------------------------------------------------------------------------
+
+codeunit 50103 "GeoLocation Test Library"
+{
+    EventSubscriberInstance = Manual;
+
+    /// <summary>
+    /// Indicate that the geographical location is available when test is in progress.
```

Comment:
Indicates that the geographical location is available when a test is in progress.